src/controller/Controller.js

```diff
@@ -0,0 +1,55 @@
+import Car from '../model/model.js';
+import { DELIMITER } from '../constants/constant.js';
+import { Random } from '@woowacourse/mission-utils';
+import {
+  getAttempts,
+  getCarNames,
+  announceWinners,
+  displayRaceProgress,
+} from '../view/view.js';
+import { validateAttempts, validCarName } from '../utils/validator.js';
+
+class Controller {
+  constructor() {
+    this.cars = [];
+  }
+
+  async execute() {
+    const enteredCarNames = await getCarNames();
+    const splittedNames = enteredCarNames.split(DELIMITER);
+    validCarName(splittedNames);
+    this.cars = this.createCars(splittedNames);
+    const attempts = await getAttempts();
+    validateAttempts(attempts);
+
+    for (let attemptIndex = 0; attemptIndex < attempts; attemptIndex++) {
+      this.startCarRace(this.cars);
+      displayRaceProgress(this.cars);
+    }
+
+    const result = this.getWinners(this.cars);
+    announceWinners(result.join(','));
+  }
```

Comment:
지금도 책임 소재에 맞게 메서드를 잘 분리해 주셨다고 생각해요! 여기서 조금더 수정해본다면 execute를 약간 더 다듬어 볼 수 있지 않을까요? 이렇게 다듬어보는건 어떨까 제안드려봅니다

```suggestion
  async execute() {
  const splittedNames = await this.getValidatedCarNames();
  this.cars = this.createCars(splittedNames);

  const attempts = await this.getValidatedAttempts();
  this.executeRaceRounds(attempts);

  const winners = this.getWinners(this.cars);
  announceWinners(winners.join(','));
}

async getValidatedCarNames() {
  const enteredCarNames = await getCarNames();
  const splittedNames = enteredCarNames.split(DELIMITER);
  validCarName(splittedNames);
  return splittedNames;
}

async getValidatedAttempts() {
  const attempts = await getAttempts();
  validateAttempts(attempts);
  return attempts;
}
```

Reply:
제안해주신 코드가 더 깔끔하고 보기 좋은 것 같아요! 
execute 함수 내부에서의 엔터 간격도 가독성에 큰 영향을 주는 것 같아요
다음 번 과제때 참고해서 구현해볼게요! 감사합니다 👍👍